pandas/core/frame.py

```diff
@@ -3101,8 +3101,10 @@ def query(self, expr, inplace=False, **kwargs):
 
             .. versionadded:: 0.25.0
 
-            You can refer to column names that contain spaces by surrounding
-            them in backticks.
+            You can refer to column names that contain spaces or operators by
```

Comment:
likely this would need to have a versioncahnged 1.0.0 here as well

Reply:
Should I specify what changed in the versions like this: 

```
            You can refer to column names that contain spaces or operators by
            surrounding them in backticks. This way you can also escape
            names that start with a digit, or are a Python keyword. Basically
            when it is not valid Python identifier.

            For example, if one of your columns is called ``a a`` and you want
            to sum it with ``b``, your query should be ```a a` + b``.

            .. versionadded:: 0.25.0: Backtick quoting introduced.

            .. versionadded:: 1.0.0: Expanding functionality for more than only spaces.
```

Reply:
sure